test/JDBC/expected/sys-table_types_internal-vu-prepare.out

```diff
@@ -0,0 +1,11 @@
+DROP TYPE IF EXISTS table_types_internal_test1;
```

Comment:
Also add test cases to test cross database/schema references for sys.table_types_internal

Reply:
added the requested tests.